src/Sulu/Bundle/MediaBundle/Media/ImageConverter/ImagineImageConverter.php

```diff
@@ -382,7 +398,13 @@ private function getCropParameters(ImageInterface $image, $formatOptions, array
      */
     private function modifyAllLayers(ImageInterface $image, callable $modifier)
     {
-        if (count($image->layers())) {
+        try {
+            $layers = $image->layers();
+        } catch (\Exception $exception) {
```

Comment:
Is there no better exception to catch?

Reply:
have changed it to `Imagine\Exception\RuntimeException `